js-packages/meep-platform-ctrl-client/test/api/UserAuthenticationApi.spec.js

```diff
@@ -93,6 +93,24 @@
           done();
         });
       });
+      describe('refreshUser', function() {
+        it('should call refreshUser successfully', function(done) {
+          // TODO: uncomment refreshUser call
```

Comment:
Unclear why this is commented.

Reply:
This is unmodified, swagger-generated code.